README.md

```diff
@@ -124,7 +124,7 @@ as this is the only environment that is supported. The only distro that is used
 is used in the [Community managed DockerHub image](https://hub.docker.com/p/apache/airflow) is
 `Debian Bookworm`. We also have support for legacy ``Debian Bullseye`` base image if you want to build a
 custom image but it is deprecated and option to do it will be removed in the Dockerfile that
-will accompany Airflow 2.9.3 so you are advised to switch to ``Debian Bookworm`` for your custom images.
+will accompany Airflow 2.10.0 so you are advised to switch to ``Debian Bookworm`` for your custom images.
```

Comment:
```suggestion
will accompany Airflow 2.9.3 so you are advised to switch to ``Debian Bookworm`` for your custom images.
```

Reply:
I think we ave not technically removed support for Bookworm for 2.10 - but we indeed could.

Reply:
I already implemented it and backported that removal to 2.10 - once we release it in. 2.10.1 it will become official in the image docs that bullseye is dropped in 2.10. 

It was really all about dockerfile supporting this https://github.com/apache/airflow/pull/41568#pullrequestreview-2244394827 so we should be fine in clarifying it's not there any more